.gitignore

```diff
@@ -108,3 +108,12 @@ storybook-static
 
 # OS Files
 .DS_Store
+
```

Comment:
Turns out I'm on the latest version of `yarn` which hasn't been used by this project yet so I was getting _tons_ of added files from `yarn`'s cache. This ensures the latest `yarn` doesn't cause `git` annoyances in the future 😄